InteractiveHtmlBom/ecad/fusion_eagle.py

```diff
@@ -0,0 +1,618 @@
+import io
+import math
+import os
+import string
+import zipfile
+from datetime import datetime
+from xml.etree import ElementTree
+
+from .common import EcadParser, Component, BoundingBox
+from .svgpath import Arc
+from ..core.fontparser import FontParser
+
+
+class FusionEagleParser(EcadParser):
+    TOP_COPPER_LAYER = '1'
+    BOT_COPPER_LAYER = '16'
+    TOP_PLACE_LAYER = '21'
+    BOT_PLACE_LAYER = '22'
+    TOP_NAMES_LAYER = '25'
+    BOT_NAMES_LAYER = '26'
+    DIMENSION_LAYER = '20'
+    TOP_DOCU_LAYER = '51'
+    BOT_DOCU_LAYER = '52'
+
+    def __init__(self, file_name, config, logger):
+        super(FusionEagleParser, self).__init__(file_name, config, logger)
+        self.config = config
+        self.font_parser = FontParser()
+        self.min_via_w = 1e-3
+        self.pcbdata = {
+            'drawings': {
+                'silkscreen': {
+                    'F': [],
+                    'B': []
+                },
+                'fabrication': {
+                    'F': [],
+                    'B': []
```

Comment:
This is never populated. Weren't docu layers supposed to be here. Right now the only thing they are used for is footprints bbox.

Reply:
Yeah.  Typically in Eagle there's hardly anything in the *Docu layers so the fabrication layer is pretty useless by itself, but I'll go ahead and add those to `fabrication` instead of `silkscreen` as discussed.